dgraphtest/cluster.go

```diff
@@ -263,6 +269,21 @@ func (hc *HTTPClient) RunGraphqlQuery(params GraphQLParams, admin bool) ([]byte,
 	return gqlResp.Data, nil
 }
 
+// RunDqlQuery makes a dql query to query endpoint
+func (hc *HTTPClient) RunDqlQuery(params DqlParams) ([]byte, error) {
```

Comment:
I feel like the content type is always going to be application/dql. If that is the case, we can get rid of the function `PostDqlQuery`, the type `DqlParams`. We will then only take body as an argument here and hard code the content type.